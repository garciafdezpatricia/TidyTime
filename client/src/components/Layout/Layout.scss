@use "../../../styles/variables" as var;

.layout {
    display: flex;
    // flex-direction: row; default value
    color: #000000;
    height: 100vh;
}