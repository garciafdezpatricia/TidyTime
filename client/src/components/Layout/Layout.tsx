// SideMenu.tsx
import React from 'react';

const SideMenu: React.FC<{ children: React.ReactNode }> = ({ children }) => {
    // Implementaci√≥n de SideMenu
    return (
        <div className="layout">
            {children}
        </div>
    );
};

export default SideMenu; // Exportar como predeterminado
