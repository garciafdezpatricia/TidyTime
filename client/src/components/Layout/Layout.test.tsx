import { render } from "@testing-library/react";
import '@testing-library/jest-dom';
import Layout from '@/src/components/Layout/Layout';

describe('Layout', () => {
    it('renders without crashing', () => {
      const { container } = render(<Layout><></></Layout>);
      expect(container).toBeInTheDocument();
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<Layout><></></Layout>);
        expect(asFragment()).toMatchSnapshot();
      });
  });