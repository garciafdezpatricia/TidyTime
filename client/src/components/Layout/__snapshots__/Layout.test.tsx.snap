// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="layout"
  />
</DocumentFragment>
`;
